include/oneapi/tbb/task_arena.h

```diff
@@ -491,6 +617,11 @@ using detail::d1::max_concurrency;
 using detail::d1::isolate;
 
 using detail::d1::enqueue;
+
+#if __TBB_PREVIEW_PARALLEL_PHASE
+using detail::d1::start_parallel_phase;
+using detail::d1::end_parallel_phase;
```

Comment:
Should not we introduce `scoped_parallel_phase` for `this_task_arena` as well?

Reply:
Oh, I see that it is already covered by the open questions in the RFC and I guess it should not be resolved in the initial implementation.
Feel free to resolve this conversation.